data.json

```diff
@@ -8,6 +8,15 @@
     "dateAdded": "2018-02-18T06:03:00.000Z",
     "tags": ["clothing", "device"]
   },
+  {
+    "name": "Alligator.io",
+    "difficulty": "hard",
+    "description": "Write a blog post on frontend development topics, get some cool Alligator stickers",
+    "reference": "https://alligator.io/write-for-us/",
+    "image": "https://pbs.twimg.com/profile_images/622200384455442432/hf71Z251_400x400.png",
+    "dateAdded": "2018-12-15T06:03:00.000Z",
```

Comment:
This should be `2018-12-09T07:08:00.000Z` since we (devSwag) were notified then

Reply:
@vikaspotluri123, hello! I fixed the timestamp. Can you please tell me where you see it? I see that the issue was opened 7 days ago, is that what you're referring to? 
:)

Reply:
Yep, that's exactly right!